templates/collection.yaml

```diff
@@ -197,6 +198,14 @@ Parameters:
      AllowedValues:
       - true
       - false
+  TrailExcludeManagementEventSources:
+    Type: String
```

Comment:
Use `CommaDelimitedList`

* Not sensitive to whitespace
* Avoids the need to split later
* Allows you to set restrictions on each list entry

Reply:
thanks!